.github/workflows/pages.yml

```
@@ -142,7 +142,7 @@ jobs:
         run: npm run build:ci
 
       - name: Run E2E tests
-        if: github.event_name == 'pull_request' || github.event_name == 'merge_group'
+        if: 'false'
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)